entity-framework/core/managing-schemas/migrations/operations.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Custom Migrations Operations - EF Core
+description: Managing custom and raw SQL migrations for schema management with Entity Framework Core
```

Comment:
Should be consistent in usage of schema vs database schema.

Reply:
Went with database schema (description is all about verboseness :trollface:)